### ISSUE TITLE
What is the accuracy of this method?

### ISSUE THREAD

**Issue by Gaozhongpai** [XuyangBai/KPConv.pytorch#1]
I have run this code and I got around 82% accuray. May I ask what is the accuracy when you run the code?

**Comment by XuyangBai** [comment]
Hi @Gaozhongpai  I got 97% accuracy for training set and 87% accuracy for validation set.

**Comment by Gaozhongpai** [comment]
> Hi @Gaozhongpai I got 97% accuracy for training set and 87% accuracy for validation set.

Thank you very much. I realized that the code is relatively slow now. Have you considered wrapping the 'batch_grid_subsampling' as a PyTorch module, which may increase the speed?

**Comment by XuyangBai** [comment]
Hi @Gaozhongpai  I didn't wrap the 'batch_grid_subsampling' as a PyTorch module because I found that in the origin cpp implementation, unlike the batch_find_neighbors function, the [batch_grid_subsampling](https://github.com/HuguesTHOMAS/KPConv/blob/935a2bd75775b07eda8cecbacf2d91b2dd7cbf8d/tf_custom_ops/tf_subsampling/grid_subsampling/grid_subsampling.cpp#L101) function just do grid subsampling for each point cloud sequentially and then stack then together,  so I think it may not affect the speed too much for my current implementation.

But yes the code is slow now, I will try to find the most time consuming block in the few days.

**Comment by tchaton** [comment]
Hey @Gaozhongpai  @XuyangBai,

We are integrating KPConv under this repo: https://github.com/nicolas-chaulet/deeppointcloud-benchmarks

**Comment by XuyangBai** [comment]
Hi @tchaton  Good Job ! I will try it : )
